### ISSUE TITLE
How do I create a empty new change using the plugin?

### ISSUE THREAD

**Issue by Mystic-Mirage** [uwolfer/gerrit-intellij-plugin#360]
I need to create an empty (draft) change without push anything.
Usually, I'm using "Create Change" button on the project's page in the "Project Commands" section.
Is there a way to do this using the plugin?

**Comment by uwolfer** [comment]
No, this is not supported. Can you elaborate what your workflow is when you do this?

**Comment by Mystic-Mirage** [comment]
My workflow is:
1. Create an empty draft change via UI: fill target branch, topic, and description (commit message)
2. Checkout that change in Gerrit plugin. This creates a new branch 'review/{name}/{topic}'
3. Do stuff, commit-with-amend
4. Push to Gerrit

**Comment by uwolfer** [comment]
@Mystic-Mirage: Ok, I see.

I think you could get the same result by checking out a new branch in IntelliJ, commit, and push to Gerrit (which creates a new change in Gerrit). And after that do your workflow you are used to. Is there a reason why you don't do it like that?

**Comment by Mystic-Mirage** [comment]
I used to use the flow you've described previously and it doesn't suites me. When I name a local branch with a custom name, Push to Gerrit doesn't detect remote branch properly. It fills the branch field with the local branch name which is totally unwanted. When I checkout existing change, Push to Gerrit detects the branch to push correctly (master/beta/etc).

**Comment by uwolfer** [comment]
> When I name a local branch with a custom name, Push to Gerrit doesn't detect remote branch properly.

That should work fine when you are using this plugin to push to Gerrit. If it does not in your case, please report it. Just make sure you create a local branch based on the "correct" remote branch (e.g. origin/master).
